### DISCUSSION TITLE
ImageGlass 9.0.10.201 - Happy Lunar New Year!

### DISCUSSION THREAD

**Discussion by d2phap** [d2phap/ImageGlass#1833] [General]
<img width="640" alt="ImageGlass 9.0.9.1230" src="https://raw.githubusercontent.com/ImageGlass/releases/main/screenshots/v9.0/9.0_b1.webp">

[![Downloads](https://img.shields.io/github/downloads/d2phap/imageglass/9.0.10.201/total?color=%232A7C91&label=downloads&style=for-the-badge)](https://imageglass.org/release/imageglass-9-0-10-201-50)

Please support this project by making a donation or purchasing the Microsoft Store version:
<a href="https://www.microsoft.com/store/productId/9N33VZK3C7TH?launch=true&mode=full&cid=GitHubRelease&referrer=appbadge&source=GithubD2phap">
  <img height="69" src="https://user-images.githubusercontent.com/3154213/197671083-4099d80f-07fc-4af9-b80d-6357f0968347.svg" />
</a>

## 🪁 System Requirements
- 🆕 Windows 10/11 64-bit, version 1809 (build 17763) or later
- [.NET Desktop Runtime 8 (64-bit)](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
- [WebView2 Runtime 64-bit v119.0.2151 or later](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section)

## 📝 Changelog (since [ImageGlass 8.11](https://github.com/d2phap/ImageGlass/releases/tag/8.10.9.27))
Please read the blogpost: https://imageglass.org/news/introducing-the-new-imageglass-version-9-88

### 🔮 Improvements:
- Added `EnableImageAsyncLoading` setting (experimental) in `igconfig.json` file that allows users to switch back to synchronous image loading as same as version 8 (#1775)
- Added new parameter `--per-machine` to command `set-default-viewer` in `igcmd.exe` to support setting default photo viewer at machine level (#1828)
- Added Exit option to the context menu (#1322)
- Added <kbd>PageUp</kbd>, <kbd>PageDown</kbd> as default hotkeys for view next/previous image
- ImageGlass Store only: Clicking "Update" button in Check for Update dialog now opens ImageGlass on Microsoft Store
- Updated icon for Viewer, Toolbar and Gallery in sidebar of the App settings
- Shows message "Slideshow paused" when slideshow ends (#1805)
- Shows more details on Unhandled Exception popup: MagickNET, WebView2 version.
- Shows more details if users run ImageGlass in Debug mode: DPI, Screen resolution, WebView2 version, Renderer engine.

### 🐛 Bugfixes:
- Fixed an issue where "VerticalBlankTicker" error appeared on some monitors (#1814, #1797) and Windows Server 2019 (#1771)
- Fixed an issue where pressing <kbd>Win</kbd> key caused image zoomed out for AZERTY keyboard layout (#1769)
- Fixed an issue where users could not save the clipboard image if the image list was empty (#1801)
- Fixed an issue where saving HEIC for other formats caused "Unsupport image format" popup (#1822)
- Fixed an issue where the crop selection reset after saving the image (#1825)
- Attempted to fix an issue where Crop button triggered "Null Reference Exception" (#1758)
- Fixed an issue where the Background color in App settings > Appearance tab did not show and load correctly
- Fixed an issue where clicking on "Get extension icon packs" link in App settings > File type associations tab did not open Extension icon packs site

**Full Changelog**: https://github.com/d2phap/ImageGlass/compare/9.0.9.1230...9.0.10.201

<hr /><em>This discussion was created from the release <a href='https://github.com/d2phap/ImageGlass/releases/tag/9.0.10.201'>ImageGlass 9.0.10.201 - Happy Lunar New Year!</a>.</em>

**Comment by username279** [comment]
Very nice program, thank you for it!
I wanted to ask, does the program not support Thumbnail Preview for .webp images? Just the other images have Thumbnail Preview, but .webp images only icon

**Reply by d2phap** [reply]
IG does not provide shell extension for thumbnail preview. You can install sagethumb instead

**Comment by HumanMus1c** [comment]
Issue:
The window's position will automatically re-center itself by switching image after turn on "keep window always on top", This is really annoying me. Please make it not re-center itself automatically while the "keep window always on top" been turn on. Thanks.

And, move "keep window always on top" option below or between “Window Fit” "Frameless" "Full Screen" "Slide Show" would be better.
![image](https://github.com/d2phap/ImageGlass/assets/46023154/56baf9ee-c0fc-4802-b028-9385c4f2cfc0)
